src/Sarif.Viewer.VisualStudio/Sarif/Location.Extensions.cs

```diff
@@ -40,5 +41,61 @@ public static LocationModel ToLocationModel(this Location location, Run run, int
 
             return model;
         }
+
+        public static string ExtractSnippet(this Location location, Run run, FileRegionsCache fileRegionsCache)
+        {
+            var physicalLocation = location.PhysicalLocation;
+            var artifactLocation = location.PhysicalLocation?.ArtifactLocation;
+            var region = location.PhysicalLocation?.Region;
+            var uri = location.PhysicalLocation?.ArtifactLocation?.Uri;
```

Comment:
can you use the object itself, since we can't know based on the usage what is it. Later, i will bring the stylecop rules from other repositories to this one.

Reply:
fixed